### PR TITLE
Bank account verification modal info

### PR DESCRIPTION
**Pull request by koko57**
### Fixed Issues

$ https://github.com/Expensify/App/issues/49472
PROPOSAL:

### Tests

(if you have already a workspace with Expensify Card enabled and you can issue a new card there, you can skip steps 1-4)
1. Go to a workspace, enable Expensify Card in More Features
2. Go to Expensify Card, click Issue New Card, add a new bank account if you don't have any
3. Choose the bank account from the list
5. After successful verification you should the Empty State screen
6. Repeat steps 1-3, choose the same bank account as in the previous workspace
7. You should the Empty State screen, but with different copier (refer to this comment) and the buttons "Issue Card" and "Settings" should not be displayed.

- [x] Verify that no errors appear in the JS console

### QA Steps

(if you have already a workspace with Expensify Card enabled and you can issue a new card there, you can skip steps 1-4)
1. Go to a workspace, enable Expensify Card in More Features
2. Go to Expensify Card, click Issue New Card, add a new bank account if you don't have any
3. Choose the bank account from the list
5. After successful verification you should the Empty State screen
6. Repeat steps 1-3, choose the same bank account as in the previous workspace
7. You should the Empty State screen, but with different copier (refer to this comment) and the buttons "Issue Card" and "Settings" should not be displayed

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
        - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

Simulator Screenshot - iPhone 15 Pro - 2024-10-25 at 19 38 19

</details>

<details>
<summary>iOS: mWeb Safari</summary>

Simulator Screenshot - iPhone 15 Pro - 2024-10-25 at 19 39 56

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

Screenshot 2024-09-20 at 20 18 01
Screenshot 2024-09-20 at 20 13 13

</details>

<details>
<summary>MacOS: Desktop</summary>

Screenshot 2024-10-25 at 19 30 14

</details>

### CODE REVIEW

File: src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx [hunk 1]
```diff
@@ -43,7 +44,15 @@ function WorkspaceExpensifyCardListPage({route, cardsList}: WorkspaceExpensifyCa
...
     const policy = usePolicy(policyID);
+    const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyID);
     const [personalDetails] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST);
+    const [cardSettings] = useOnyx(`${ONYXKEYS.COLLECTION.PRIVATE_EXPENSIFY_CARD_SETTINGS}${workspaceAccountID}`);
+
+    const paymentBankAccountID = cardSettings?.paymentBankAccountID ?? 0;
+    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+
+    // TODO: change approvedBy check for more accurate verification status
+    const isBankAccountVerified = bankAccountsList?.[paymentBankAccountID]?.accountData?.approvedBy === 'Automatic';
```
Comment (koko57) [inline src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx:55]: How should I check if the account is verified or not?

File: src/languages/es.ts [hunk 2]
```diff
@@ -2927,6 +2927,8 @@ export default {
         expensifyCard: {
             issueAndManageCards: 'Emitir y gestionar Tarjetas Expensify',
             getStartedIssuing: 'Empieza emitiendo tu primera tarjeta virtual o física.',
+            verificationInProgress: 'Verificación en curso...',
+            verifyingTheDetails: "We're verifying a few details. Concierge will let you know when Expensify Cards are ready to issue.",
```
Comment (koko57) [inline src/languages/es.ts:2931]: asking for approving Spanish translation

File: src/pages/workspace/expensifyCard/EmptyCardView.tsx [hunk 3]
```diff
@@ -13,13 +13,24 @@ import useWindowDimensions from '@hooks/useWindowDimensions';
...
 
-function EmptyCardView() {
+type EmptyCardViewProps = {
+    /** Title of the empty state */
+    title: string;
+
+    /** Subtitle of the empty state */
+    subtitle: string;
+
+    /** Whether the header buttons be displayed */
```
Comment (mountiny) [inline src/pages/workspace/expensifyCard/EmptyCardView.tsx:23]:
```suggestion
    /** Whether the header buttons are displayed */
```

File: src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx (hunk 1 above)
Comment (mountiny) [reply src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx:55]: I need to double check this, I think the bank account state wont be open but verifying but I am not 100% sure

Comment (mountiny) [reply src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx:55]: Ok yeah I think it should be in `VERIFYING` state so you need to check the state of the linked bank account is that

**Comment by mountiny** [comment]
Shared the instructions for how to create the verifying bank account with you in our dm

File: src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx (hunk 1 above)
Comment (koko57) [reply src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx:55]: @mountiny so can I just check if the bank account is in OPEN state? For other states like PENDING I guess we don't want to display the buttons either

Comment (mountiny) [reply src/pages/workspace/expensifyCard/WorkspaceExpensifyCardListPage.tsx]: @koko57 that is a fair assumption, only the open state should be able to create cards

**Comment by dubielzyk-expensify** [comment]
This is my bad (as I did the mocks), but I wonder when I see it now if we should reduce the height of the illustration/video above "Verification in progress". Our empty states have 220px height, but that feels too small though I see that in Figma @shawnborton commented on using the empty state component here and change the illustration like so:

image

Keen to hear if @Expensify/design agrees.

**Comment by koko57** [comment]
@dubielzyk-expensify ok, so please confirm if I should make these changes

**Comment by shawnborton** [comment]
I would love to stay consistent with other empty states. I think they are all 220px right?

**Comment by dubielzyk-expensify** [comment]
Yep. 220px.

**Comment by koko57** [comment]
@dubielzyk-expensify @shawnborton ok, so for the Verification state we're changing the illustration for this one: 
Screenshot 2024-10-28 at 17 37 34
and the height should be 220px.

Should the height of the illustration also change to 220px for the empty card state modal?
It'll look like this after the changes:
Screenshot 2024-10-28 at 17 36 49

**Comment by dubielzyk-expensify** [comment]
Might have to adjust, but the container should be 220. The illustration height should be 162 according to [Figma](https://www.figma.com/design/9J4v6CzMVLQbzXMnXC5Izq/%23wave-collect%3A-Workspace-Card-Feeds?node-id=6054-18679&t=hx3AxB2Za0LRCAtp-4) :) 
CleanShot 2024-10-28 at 20 42 49@2x

I think we have the illustration already checked in too

**Comment by dubielzyk-expensify** [comment]
And as for your question, I think we should just replace that illustration with something else. Let me re-do it 👍

**Comment by dubielzyk-expensify** [comment]
Can we use the Expensify Cards illustration we're already using? 
CleanShot 2024-10-28 at 20 45 51@2x

**Comment by shawnborton** [comment]
> The illustration height should be...

I think that depends on the illustration itself though, right? Since it changes depending on the context. 

And it sounds like for this particular project, we want to use this illustration right? Because it communicates that verification is happening... 
CleanShot 2024-10-28 at 11 50 58@2x

**Comment by koko57** [comment]
> Can we use the Expensify Cards illustration we're already using? CleanShot 2024-10-28 at 20 46 25@2x

So should I change this illustration for verified empty state for this one?

**Comment by shawnborton** [comment]
I don't think we want to change the illustration personally, I think if the verification is in process, we want to use what Jon has [here](https://github.com/Expensify/App/pull/49536#issuecomment-2440555685).

cc @Expensify/design for more thoughts though!

**Comment by koko57** [comment]
ok, so I will change only the Verification Modal image for now. If you decide to do some changes to the Empty State modal just ping me! 😊

**Comment by dannymcclain** [comment]
Totally agree Shawn!

**Comment by koko57** [comment]
Screenshot 2024-10-28 at 21 15 36
Screenshot 2024-10-28 at 21 15 24

already applied

**Comment by shawnborton** [comment]
Lovely!

**Comment by dubielzyk-expensify** [comment]
Looks great!

**Comment by koko57** [comment]
@DylanDylann ready for review

**Comment by DylanDylann** [comment]
I will try to complete my reviewing for these issues on Wed

File: src/hooks/useEmptyViewHeaderHeight/index.ts [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import {BUTTON_HEIGHT, BUTTON_MARGIN, HEADER_HEIGHT} from './const';
 
-function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean): number {
-    return isSmallScreenWidth ? HEADER_HEIGHT + BUTTON_HEIGHT + BUTTON_MARGIN : HEADER_HEIGHT;
+function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean, areHeaderButtonsDisplayed: boolean): number {
```
Comment (DylanDylann) [inline src/hooks/useEmptyViewHeaderHeight/index.ts:3]:
```suggestion
function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean, isHeaderButtonsDisplayed: boolean): number {
```

File: src/hooks/useEmptyViewHeaderHeight/index.ios.ts [hunk 5]
```diff
@@ -1,10 +1,11 @@
 import useSafeAreaInsets from '@hooks/useSafeAreaInsets';
 import {BUTTON_HEIGHT, BUTTON_MARGIN, HEADER_HEIGHT} from './const';
 
-function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean): number {
+function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean, areHeaderButtonsDisplayed: boolean): number {
```
Comment (DylanDylann) [inline src/hooks/useEmptyViewHeaderHeight/index.ios.ts:4]:
```suggestion
function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean, isHeaderButtonsDisplayed: boolean): number {
```

**Comment by koko57** [comment]
@DylanDylann yes - translations confirmed: https://callstack-hq.slack.com/archives/C01GTK53T8Q/p1730202198842939?thread_ts=1730196432.149739&cid=C01GTK53T8Q

File: src/hooks/useEmptyViewHeaderHeight/index.ts (hunk 4 above)
Comment (koko57) [reply src/hooks/useEmptyViewHeaderHeight/index.ts:3]: it's plural - why hould I change that 😅

File: src/hooks/useEmptyViewHeaderHeight/index.ios.ts (hunk 5 above)
Comment (koko57) [reply src/hooks/useEmptyViewHeaderHeight/index.ios.ts:4]: Screenshot 2024-10-30 at 12 31 13
 here we also have plural, nobody protested 😄

Comment (koko57) [reply src/hooks/useEmptyViewHeaderHeight/index.ios.ts:4]: another examples 😃 
Screenshot 2024-10-30 at 12 32 24

File: src/hooks/useEmptyViewHeaderHeight/index.ts (hunk 4 above)
Comment (DylanDylann) [reply src/hooks/useEmptyViewHeaderHeight/index.ts:3]: Ahh sorry

**Comment by DylanDylann** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>





Screenshot 2024-10-30 at 13 31 16

</details>

<details>
<summary>Android: mWeb Chrome</summary>



Screenshot 2024-10-30 at 12 33 07

</details>

<details>
<summary>iOS: Native</summary>

Screenshot 2024-10-30 at 13 31 10



</details>

<details>
<summary>iOS: mWeb Safari</summary>

Screenshot 2024-10-30 at 12 32 57



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



Screenshot 2024-10-30 at 12 29 14


</details>

<details>
<summary>MacOS: Desktop</summary>



Screenshot 2024-10-30 at 12 34 09


</details>

**Review by DylanDylann — APPROVED** [review]

**Comment by DylanDylann** [comment]
Discover new bug (not relate to this PR): https://expensify.slack.com/archives/C05LX9D6E07/p1730270492636799?thread_ts=1728966082.429659&cid=C05LX9D6E07

**Review by mountiny — APPROVED** [review]
Thanks!

**Comment by mountiny** [comment]
I dont think this performance regression could be legit on this pr

**Pull request merged**